index.html

```diff
@@ -456,17 +456,6 @@ <h2><dfn>observe()</dfn> method</h2>
             <var>relevantGlobal</var>, with <a>observer</a> set to the
             <a>context object</a> and <a>options list</a> set to a list
             containing <var>options</var> as its only item.</li>
-            <li>If <var>options</var>'s <a>buffered</a> flag is set:
```

Comment:
@plehegar - can you clarify why we need to remove the buffered flag here?

Reply:
I believe the goal was to remove the buffered flag in #114. I mixed things when I did the merge.

Reply:
The goal was to remove it from L2, but not from gh-pages